jazz-build-module/scm-module.groovy

```diff
@@ -0,0 +1,179 @@
+#!groovy?
+import groovy.json.JsonOutput
+import groovy.transform.Field
+import groovy.json.JsonSlurper
+/*
+* Module that handles managing projects (create, delete) in the user's preferred scm
+*/
+
+def scm_core_services_endpoint
+def scm_user_services_api_endpoint
+def scm_user_services_clone_url
+def scm_webhook_api
+def scm_branch_permission_api_endpoint
+
+def serviceonboarding_repo
+def scm_config
+@Field def scm_protocol	= "http://"
```

Comment:
Should this be based on the SCM we chose at the time installation (For example, choosing an existing SCM implementation which is SSL only) ?

Reply:
Sure, that will part of another story as this functionality is as-is.